integrations/standalone/tsconfig.json

```diff
@@ -1,20 +1,8 @@
 {
+  "extends": "../../config/tsconfig.integration.json",
   "compilerOptions": {
-    "incremental": true,
-    "target": "esnext",
     "types": ["vite/client", "node"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
+    "noImplicitOverride": false,
```

Comment:
Had do change that for the integration, there is a lot of test code which does not declare the override explicitly. May be we should change that?

Reply:
why is the testcode a problem for the integration?
But yes I'm open for this change

Reply:
In my opinion in the end the noImplicitOverride should be removed here, and so true should be used for this flag as on the tsconfig.base. But at the moment a lot of test code does compile with this flag set to true.

Reply:
you mean playwright code? or jest code?